code/modules/organs/robolimbs.dm

```diff
@@ -18,6 +18,7 @@ var/global/datum/robolimb/basic_robolimb
 	var/desc = "A generic unbranded robotic prosthesis." // Seen when examining a limb.
 	var/icon = 'icons/mob/human_races/ipc/robotic.dmi'       // Icon base to draw from.
 	var/unavailable_at_chargen                           // If set, not available at chargen.
+	var/lifelike                                         // If set, appears organic.
```

Comment:
```suggestion
	var/lifelike = FALSE                                         // If set, appears organic.
```

Reply:
Pretty sure null is a falsey value anyway, so there's no need to explicitly initialise it.

Reply:
There's no need, but it does make it clearer what it is